worker/src/main.rs

```diff
@@ -104,8 +106,7 @@ async fn rebuild(client: &Client, config: &config::ConfigFile) -> Result<()> {
         JobAssignment::Rebuild(rb) => {
             info!("Starting rebuild of {:?} {:?}",  rb.package.name, rb.package.version);
             let distro = rb.package.distro.parse::<Distro>()?;
-            let rebuild = match spawn_rebuilder_script_with_heartbeat(client, &distro, &rb, config).await {
-                Ok(res) => {
+            let rebuild = match spawn_rebuilder_script_with_heartbeat(&client, &distro, &privkey, &rb, config).await {                Ok(res) => {
```

Comment:
I believe at least `Ok` can go to the next line

Reply:
Oof editor :"))))